public-ui/src/app/data-browser/views/intro-vids/intro-vids_react/intro-vids_react.component.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { FunctionComponent } from 'react';
+import { Video } from '../../../services/video.service';
+
+
+export const IntroVidReactComponent: FunctionComponent<Video> =
+    (props) => {
+        return <span>
```

Comment:
If you don't need the extra `<span>` here, you can use `<React.Fragment>` and it will just render everything inside of it.